database/seed.js

```diff
@@ -0,0 +1,144 @@
+    reviews: 1688,
+    verified: true,
+    superhost: true,
+    superhostIcon: 'https://www.pinpng.com/pngs/m/13-133921_responsive-website-by-pelican-design-consultants-airbnb-superhost.png',
+    coHost: [3, 4],
+    joined_at: '2015-01-14T05:05:26.037Z',
+    languages: 'English, Korean, German',
+    responseTime: 'within an hour',
+    responseRate: 98,
+    location: 'Sunnyvale, CA',
+  provided: {
+    email: true,
+    phone: true,
+    id: true
+  },
+    avatarUrl: 'https://source.unsplash.com/1600x900/?portrait'
+  },
+  {
+    id: 3,
+    name: 'Tribe',
+    description: 'Tribe is a lifestyle brand, here to bring the most refined properties in the city.',
+    reviews: 127,
+    verified: true,
+    superhost: false,
+    coHost: [2],
+    joined_at: '2019-11-18T05:05:26.037Z',
+    languages: 'English',
+    responseTime: 'within an hour',
+    responseRate: 92,
+    location: 'San Jose, CA',
+  provided: {
+    email: false,
+    phone: true,
+    id: false
+  },
+    avatarUrl: 'https://source.unsplash.com/1600x900/?portrait'
+  },
+  {
+    id: 4,
+    name: 'Kathy & Craig',
+    description: 'We met and became friends 25 years ago, then about 9 years ago (single again after marriages and families with other people) decided that WE should be a couple. Weve been happily married for 5 years now, and our Airbnb rental is on the property of the Sunnyvale house we bought together and lovingly renovated. Craig is from Baton Rouge, Louisiana (still says y\'all despite 25 years away from the South). He\'s a tech writer at Cisco. He loves music, reading, vintage Mustangs, and recently has taken up video gaming. I\'m from Pittsburgh, PA and am a Program Manager at Symantec Corp. I love cooking, gardening, decorating and redecorating, painting and reading.We love our house and neighborhood and want everyone who stays with us to to feel comfortable and welcome.',
+    duringStay: 'You will enjoy staying in this property',
+    reviews: 16,
+    verified: true,
+    superhost: true,
+    superhostIcon: 'https://www.pinpng.com/pngs/m/13-133921_responsive-website-by-pelican-design-consultants-airbnb-superhost.png',
+    coHost: [2],
+    joined_at: '2011-01-14T05:05:26.037Z',
+    languages: 'English, Spanish',
+    responseTime: 'within an hour',
+    responseRate: 100,
+    location: 'Honolulu, HI',
+  provided: {
+    email: true,
+    phone: false,
+    id: true
+  },
+    avatarUrl: 'https://source.unsplash.com/1600x900/?portrait'
+  }
+
+
+];
+var randomLocation = ['San Jose, CA', 'New Deli, India', 'Moscow, Russia', 'Paris, France', 'Yerevan, Armenia', 'San Francisco, CA', 'Berlin, Germany', 'Rome, Italy', 'Napa, CA'];
+var randomLanguage = ['English', 'Chinese', 'Spanish', 'Hindi', 'Arabic', 'PORTUGUESE', 'Russian'];
+var randomResponse = ['within an hour', 'within a day', 'within a minute', 'within a week' ,'within 2 hours'];
+var randomBoolean = [true, false];
+
+const insertSampleData = function() {
```

Comment:
Way to go on this function @AnushAvagyan. Very cool way to randomize. I will totally adapt this to create the rest of my mock records.

Reply:
thanks, Justin